client/src/index.jsx

```diff
@@ -1,24 +1,41 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import axios from 'axios';
+import Header from './header.jsx';
+import Overview from './overview.jsx';
+import Menu from './menu.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-
+      data: {
+        header: {},
+        menus: {},
+      },
     };
   }
 
-  componentDidMount() {
+  componentWillMount() {
     axios.get('/restaurants/30590734/overview')
-    .then(data => {console.log(data)});
+    .then(data => {
+      console.log(data.data[0]);
```

Comment:
Do you need the console.log in the client code?

Reply:
Good catch!